### PR TITLE
added full schema

### PR DESCRIPTION
**Pull request by deemaagog**

### CODE REVIEW

File: static/admin/config.yml [hunk 1]
```diff
@@ -12,25 +12,32 @@ collections:
     create: true
     slug: "{{slug}}"
     fields: 
-      - {label: "Id", name: "id", widget: "string"}
       - {label: "Title", name: "title", widget: "string"}
+      - {label: "Name", name: "name", widget: "string"}
       - {label: "Description", name: "description", widget: "text"}
-      # - {label: "Priority", name: "priority", widget: "number"}
+      - {label: "KeyWords", name: "keywords", widget: "text"}
```
Comment (comerc) [inline static/admin/config.yml]: Typo Keywords

File: static/admin/config.yml [hunk 2]
```diff
@@ -12,25 +12,32 @@ collections:
     create: true
     slug: "{{slug}}"
     fields: 
-      - {label: "Id", name: "id", widget: "string"}
       - {label: "Title", name: "title", widget: "string"}
```
Comment (comerc) [inline static/admin/config.yml]: Rename label to "Id"

File: static/admin/config.yml [hunk 3]
```diff
@@ -12,25 +12,32 @@ collections:
...
+      - {label: "Name", name: "name", widget: "string"}
```
Comment (comerc) [inline static/admin/config.yml]: Rename label to "Title"

File: static/admin/config.yml [hunk 4]
```diff
@@ -12,25 +12,32 @@ collections:
...
     label: "Posts"
     folder: "src/data/posts"
     create: true
-    slug: "{{year}}-{{month}}-{{day}}-{{slug}}"
+    slug: "{{slug}}"
     fields:
-      - {label: "Template Key", name: "templateKey", widget: "hidden", default: "post"}
+      # - {label: "Template Key", name: "templateKey", widget: "hidden", default: "post"}
       - {label: "Title", name: "title", widget: "string"}
+      - {label: "Name", name: "name", widget: "string"}
```
Comment (comerc) [inline static/admin/config.yml]: Rename label to "Title"

File: static/admin/config.yml [hunk 5]
```diff
@@ -12,25 +12,32 @@ collections:
...
   - name: "posts"
     label: "Posts"
     folder: "src/data/posts"
     create: true
-    slug: "{{year}}-{{month}}-{{day}}-{{slug}}"
+    slug: "{{slug}}"
     fields:
-      - {label: "Template Key", name: "templateKey", widget: "hidden", default: "post"}
+      # - {label: "Template Key", name: "templateKey", widget: "hidden", default: "post"}
       - {label: "Title", name: "title", widget: "string"}
```
Comment (comerc) [inline static/admin/config.yml]: Rename to "Id"

File: static/admin/config.yml [hunk 6]
```diff
@@ -12,25 +12,32 @@ collections:
...
+      - {label: "Name", name: "name", widget: "string"}
+      - {label: "Is published", name: "isPublished", widget: "boolean", required: false}
```
Comment (comerc) [inline static/admin/config.yml]: Change `required: false` to `default: false`

File: static/admin/config.yml [hunk 7]
```diff
@@ -12,25 +12,32 @@ collections:
...
       - {label: "Publish Date", name: "date", widget: "datetime"}
```
Comment (comerc) [inline static/admin/config.yml]: Change "name: date" to "name: publishDate"

File: static/admin/config.yml [hunk 8]
```diff
@@ -12,25 +12,32 @@ collections:
...
-      # - label: "Tags"
-      #   name: "tags"
-      #   widget: "list"
-      #   default: ["news"]
+      # - {label: Image, name: image, widget: image}
+      - label: "Image"
+        name: "image"
+        widget: "object"
+        fields: 
+        - {label: "Image", name: "Image", widget: "image"}
```
Comment (comerc) [inline static/admin/config.yml]: Typo "name: image"

File: static/admin/config.yml [hunk 9]
```diff
@@ -12,25 +12,32 @@ collections:
...
+      - {label: "Is published", name: "isPublished", widget: "boolean", required: false}
       - {label: "Publish Date", name: "date", widget: "datetime"}
       - {label: "Description", name: "description", widget: "text"}
+      - {label: "Lead", name: "lead", widget: "text"}
       - {label: "Body", name: "body", widget: "markdown"}
-      # - label: "Tags"
-      #   name: "tags"
-      #   widget: "list"
-      #   default: ["news"]
+      # - {label: Image, name: image, widget: image}
```
Comment (comerc) [inline static/admin/config.yml]: Remove comment

File: static/admin/config.yml [hunk 10]
```diff
@@ -40,26 +47,53 @@ collections:
...
+      # - {label: "Template Key", name: "templateKey", widget: "hidden", default: "post"}
+      - {label: "Title", name: "title", widget: "string"}
+      - {label: "Name", name: "name", widget: "string"}
+      - {label: "Description", name: "description", widget: "text"}
+      - {label: "Label", name: "label", widget: "string"}
+      - label: "Image"
+        name: "image"
+        widget: "object"
+        fields: 
+        - {label: "Image", name: "Image", widget: "image"}
```
Comment (comerc) [inline static/admin/config.yml]: Typo "name: image"

File: static/admin/config.yml [hunk 11]
```diff
@@ -69,13 +103,41 @@ collections:
...
+        fields:
+          - {label: "Title", name: "title", widget: "string"}
+          - {label: "Name", name: "name", widget: "string"}
+          - {label: "Description", name: "description", widget: "text"}
           - {label: "Body", name: "body", widget: "markdown"}
+          - label: "Seo"
+            name: "seo"
+            widget: "object"
+            fields: 
+            - {label: "Seo description", name: "seoDescription", widget: "string", required: false}
```
Comment (comerc) [inline static/admin/config.yml]: - Change label: "Description"
- Change name: "description"

File: static/admin/config.yml [hunk 12]
```diff
@@ -69,13 +103,41 @@ collections:
...
-          - {label: "Template Key", name: "templateKey", widget: "hidden", default: "about-page"}
           - {label: "Title", name: "title", widget: "string"}
+          - {label: "Name", name: "name", widget: "string"}
+          - {label: "Description", name: "description", widget: "text"}
           - {label: "Body", name: "body", widget: "markdown"}
+          - label: "Seo"
+            name: "seo"
+            widget: "object"
+            fields: 
+            - {label: "Seo description", name: "seoDescription", widget: "string", required: false}
```
Comment (comerc) [inline static/admin/config.yml]: - Change label: "Description"
- Change name: "description"
- Remove required: false
- Add default: false

File: static/admin/config.yml [hunk 13]
```diff
@@ -40,26 +47,53 @@ collections:
...
-      #         - {label: Name, name: name, widget: string, default: "Emmet"}
-      #         - {label: Avatar, name: avatar, widget: image, default: "/img/emmet.jpg"}
+            searchFields: ["name", "description"]
+            valueField: "title"
+            displayFields: ["name"]  
+      - label: "Seo"
+        name: "seo"
+        widget: "object"
+        fields: 
+        - {label: "Seo description", name: "seoDescription", widget: "string", required: false}
```
Comment (comerc) [inline static/admin/config.yml]: - Change label: "Description"
- Change name: "description"
- Remove required: false
- Add default: false

File: static/admin/config.yml [hunk 14]
```diff
@@ -40,26 +47,53 @@ collections:
...
+        - {label: "Seo keyWords", name: "seoKeyWords", widget: "text", required: false}
```
Comment (comerc) [inline static/admin/config.yml]: - Change label: "Keywords"
- Change name: "keywords"
- Remove required: false
- Add default: ""

File: static/admin/config.yml [hunk 15]
```diff
@@ -40,26 +47,53 @@ collections:
...
+                  
 
+  - name: "projects"
+    label: "Projects"
+    folder: "src/data/projects"
+    create: true
+    slug: "{{slug}}"
+    fields:
+      # - {label: "Template Key", name: "templateKey", widget: "hidden", default: "post"}
+      - {label: "Title", name: "title", widget: "string"}
```
Comment (comerc) [inline static/admin/config.yml]: Rename to "Id"

File: static/admin/config.yml [hunk 16]
```diff
@@ -40,26 +47,53 @@ collections:
...
+      - {label: "Name", name: "name", widget: "string"}
```
Comment (comerc) [inline static/admin/config.yml]: Rename to "Title"

File: static/admin/config.yml [hunk 17]
```diff
@@ -40,26 +47,53 @@ collections:
...
+      - {label: "Description", name: "description", widget: "text"}
+      - {label: "Label", name: "label", widget: "string"}
+      - label: "Image"
+        name: "image"
+        widget: "object"
+        fields: 
+        - {label: "Image", name: "Image", widget: "image"}
+        - {label: "Alt", name: "alt", widget: "string"}
+      - {label: Main post, name: main post, widget: relation, collection: "posts", searchFields: ["name", "description"], valueField: "title", displayFields: ["name", "description"]}
```
Comment (comerc) [inline static/admin/config.yml]: - Fix quotes and name
- Format code to multiline

File: static/admin/config.yml [hunk 18]
```diff
@@ -40,26 +47,53 @@ collections:
...
+        # allow_add: false
+        fields:
+          - label: "Post"
+            name: "post"
+            widget: "relation"
+            collection: "posts"
+            searchFields: ["name", "description"]
+            valueField: "title"
+            displayFields: ["name", "description"]  
+      - label: "Seo"
```
Comment (comerc) [inline static/admin/config.yml]: Typo SEO

File: static/admin/config.yml [hunk 19]
```diff
@@ -40,26 +47,53 @@ collections:
...
+        name: "seo"
+        widget: "object"
+        fields: 
+        - {label: "Seo description", name: "seoDescription", widget: "string", required: false}
```
Comment (comerc) [inline static/admin/config.yml]: - Change label: "Description"
- Change name: "description"
- Remove required: false
- Add default: false

File: static/admin/config.yml [hunk 20]
```diff
@@ -40,26 +47,53 @@ collections:
...
+        - {label: "Seo keyWords", name: "seoKeyWords", widget: "text", required: false}
```
Comment (comerc) [inline static/admin/config.yml]: - Change label: "Keywords"
- Change name: "keywords"
- Remove required: false
- Add default: ""

File: static/admin/config.yml [hunk 21]
```diff
@@ -69,13 +103,41 @@ collections:
         label: "About"
         name: "about"
         fields:
-          - {label: "Template Key", name: "templateKey", widget: "hidden", default: "about-page"}
           - {label: "Title", name: "title", widget: "string"}
```
Comment (comerc) [inline static/admin/config.yml]: Rename to "Id"

File: static/admin/config.yml [hunk 22]
```diff
@@ -69,13 +103,41 @@ collections:
...
+          - {label: "Name", name: "name", widget: "string"}
```
Comment (comerc) [inline static/admin/config.yml]: Rename to Title

File: static/admin/config.yml [hunk 23]
```diff
@@ -69,13 +103,41 @@ collections:
...
+          - {label: "Description", name: "description", widget: "text"}
           - {label: "Body", name: "body", widget: "markdown"}
+          - label: "Seo"
+            name: "seo"
+            widget: "object"
+            fields: 
+            - {label: "Seo description", name: "seoDescription", widget: "string", required: false}
+            - {label: "Seo keyWords", name: "seoKeyWords", widget: "text", required: false}
```
Comment (comerc) [inline static/admin/config.yml]: - Change label: "Keywords"
- Change name: "keywords"
- Remove required: false
- Add default: ""

File: static/admin/config.yml [hunk 24]
```diff
@@ -69,13 +103,41 @@ collections:
...
       - file: "src/pages/contacts/index.md"
         label: "Contacts"
         name: "contacts"
         fields:
-          - {label: "Template Key", name: "templateKey", widget: "hidden", default: "contact-page"}
-          - {label: Title, name: title, widget: string}
+          - {label: "Title", name: "title", widget: "string"}
```
Comment (comerc) [inline static/admin/config.yml]: Rename to "Id"

File: static/admin/config.yml [hunk 25]
```diff
@@ -69,13 +103,41 @@ collections:
...
+          - {label: "Name", name: "name", widget: "string"}
```
Comment (comerc) [inline static/admin/config.yml]: Rename to "Title"

File: static/admin/config.yml [hunk 26]
```diff
@@ -69,13 +103,41 @@ collections:
...
+          - {label: "Description", name: "description", widget: "text"}
+          - {label: "Body", name: "body", widget: "markdown"}
+          - label: "Seo"
+            name: "seo"
+            widget: "object"
+            fields: 
+            - {label: "Seo description", name: "seoDescription", widget: "string", required: false}
```
Comment (comerc) [inline static/admin/config.yml]: - Add tab for list of Yaml

File: static/admin/config.yml [hunk 27]
```diff
@@ -69,13 +103,41 @@ collections:
...
+          - {label: "Title", name: "title", widget: "string"}
+          - {label: "Name", name: "name", widget: "string"}
+          - {label: "Description", name: "description", widget: "text"}
           - {label: "Body", name: "body", widget: "markdown"}
+          - label: "Seo"
+            name: "seo"
+            widget: "object"
+            fields: 
+            - {label: "Seo description", name: "seoDescription", widget: "string", required: false}
+            - {label: "Seo keyWords", name: "seoKeyWords", widget: "text", required: false}
```
Comment (comerc) [inline static/admin/config.yml]: - Change label: "Keywords"
- Change name: "keywords"
- Remove required: false
- Add default: ""

File: static/admin/config.yml [hunk 28]
```diff
@@ -40,26 +47,53 @@ collections:
...
+        name: "image"
+        widget: "object"
+        fields: 
+        - {label: "Image", name: "Image", widget: "image"}
+        - {label: "Alt", name: "alt", widget: "string"}
+      - {label: Main post, name: main post, widget: relation, collection: "posts", searchFields: ["name", "description"], valueField: "title", displayFields: ["name", "description"]}
+      - label: "Posts"
+        name: "posts"
+        widget: "list"
+        # allow_add: false
```
Comment (comerc) [inline static/admin/config.yml]: Remove comment

File: static/admin/config.yml [hunk 29]
```diff
@@ -69,13 +103,41 @@ collections:
...
+            name: "seo"
+            widget: "object"
+            fields: 
+            - {label: "Seo description", name: "seoDescription", widget: "string", required: false}
+            - {label: "Seo keyWords", name: "seoKeyWords", widget: "text", required: false}
+      - file: "src/pages/advertising/index.md"
+        label: "Advertising"
+        name: "advertising"
+        fields:
+          - {label: "Title", name: "title", widget: "string"}
```
Comment (comerc) [inline static/admin/config.yml]: Rename to "Id"

File: static/admin/config.yml [hunk 30]
```diff
@@ -69,13 +103,41 @@ collections:
...
+          - {label: "Name", name: "name", widget: "string"}
```
Comment (comerc) [inline static/admin/config.yml]: Rename to "Title"

File: static/admin/config.yml (hunk 1 above)
Comment (deemaagog) [reply static/admin/config.yml]: done

File: static/admin/config.yml (hunk 2 above)
Comment (deemaagog) [reply static/admin/config.yml]: done

File: static/admin/config.yml (hunk 3 above)
Comment (deemaagog) [reply static/admin/config.yml]: done

File: static/admin/config.yml (hunk 4 above)
Comment (deemaagog) [reply static/admin/config.yml]: done

File: static/admin/config.yml (hunk 5 above)
Comment (deemaagog) [reply static/admin/config.yml]: done

File: static/admin/config.yml (hunk 6 above)
Comment (deemaagog) [reply static/admin/config.yml]: +

File: static/admin/config.yml (hunk 7 above)
Comment (deemaagog) [reply static/admin/config.yml]: done

File: static/admin/config.yml (hunk 8 above)
Comment (deemaagog) [reply static/admin/config.yml]: done

File: static/admin/config.yml (hunk 9 above)
Comment (deemaagog) [reply static/admin/config.yml]: done

File: static/admin/config.yml (hunk 10 above)
Comment (deemaagog) [reply static/admin/config.yml]: done

File: static/admin/config.yml (hunk 11 above)
Comment (deemaagog) [reply static/admin/config.yml]: done

File: static/admin/config.yml (hunk 12 above)
Comment (deemaagog) [reply static/admin/config.yml]: done

File: static/admin/config.yml (hunk 13 above)
Comment (deemaagog) [reply static/admin/config.yml]: done

File: static/admin/config.yml (hunk 21 above)
Comment (deemaagog) [reply static/admin/config.yml]: done

File: static/admin/config.yml (hunk 22 above)
Comment (deemaagog) [reply static/admin/config.yml]: done

File: static/admin/config.yml (hunk 23 above)
Comment (deemaagog) [reply static/admin/config.yml]: done

File: static/admin/config.yml (hunk 24 above)
Comment (deemaagog) [reply static/admin/config.yml]: done

File: static/admin/config.yml (hunk 25 above)
Comment (deemaagog) [reply static/admin/config.yml]: done

File: static/admin/config.yml (hunk 27 above)
Comment (deemaagog) [reply static/admin/config.yml]: done

File: static/admin/config.yml (hunk 28 above)
Comment (deemaagog) [reply static/admin/config.yml]: +

File: static/admin/config.yml (hunk 29 above)
Comment (deemaagog) [reply static/admin/config.yml]: +

File: static/admin/config.yml (hunk 30 above)
Comment (deemaagog) [reply static/admin/config.yml]: +

**Pull request merged**